k9mail/src/main/java/com/fsck/k9/helper/HtmlConverter.java

```diff
@@ -1323,6 +1323,30 @@ private static String htmlifyMessageFooter() {
         return "</pre>";
     }
 
+    public static String wrapStatusMessage(CharSequence status) {
```

Comment:
the sole calling place has the argument as a StringBuilder, so it was either `toString()` it or accept the CharSequence, which seemed fine since we do nothing wit it but append in a StringBuilder-sequence